cockroachdb/manifests-cert-manager/client.yaml

```
@@ -4,7 +4,7 @@ metadata:
   name: &app cockroachdb-client
   annotations:
     "app.uw.systems/description": "Used to connect and query the cockroachdb databases."
-    "app.uw.systems/repos.cockroachdb-manifests": "https://github.com/utilitywarehouse/cockroachdb-manifests"
+    "app.uw.systems/repos": "https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/cockroachdb"
```

I don't think we want the `tree` structure (at least, the docs don't mention it https://wiki.uw.systems/posts/ops-level-nz4v4ka0#hqoo3-app-uw-systems-repos-id)

```suggestion
    "app.uw.systems/repos": "https://github.com/utilitywarehouse/shared-kustomize-bases/cockroachdb"
```

---

Since we have the same structure in all of the shared-manifests, I would prefer to change that in separate PR.

---

> Since we have the same structure in all of the shared-manifests, I would prefer to change that in separate PR.

I think the others might also need updating in that separate PR because it looks like the way we build the link for OpsLevel doesn't expect the `tree/<branch-name>` bit e.g. the repo on https://app.opslevel.com/services/shared-redis-master points to https://github.com/utilitywarehouse/shared-kustomize-bases/main/tree/main/redis,